php/commands/site.php

```diff
@@ -698,6 +698,60 @@ public function create_env($args, $assoc_args) {
    }
 
    /**
+    * Merge a Multidev Environment into Dev Environment
+    *
+    * ## OPTIONS
+    *
+    * [--site=<site>]
+    * : Site to use
+    *
+    * [--env=<env>]
+    * : Name of multidev to environment to merge into Dev
+    *
+    * @subcommand merge-to-dev
+    */
+    public function merge_to_dev($args, $assoc_args) {
+      $site = SiteFactory::instance(Input::site($assoc_args));
+      $site->environmentsCollection->fetch();
+
+      $multidev_ids = array_map(function($env) { return $env->id; }, $site->environmentsCollection->multidev());
+      $multidev_id = Input::env($assoc_args, 'env', "Multidev environment to merge into Dev Environment", $multidev_ids);
+      $environment = $site->environmentsCollection->get($multidev_id);
+
+      $workflow = $environment->mergeToMaster();
```

Comment:
`$workflow = $environment->mergeFromMaster(array('from_environment' => $environment->getId()));`
With the function name appropriately renamed.